composer.json

```diff
@@ -7,7 +7,8 @@
     },
     "require": {
         "filp/whoops": "^1.1 || ^2.0",
-        "zendframework/zend-diactoros": "^1.1.0"
+        "zendframework/zend-diactoros": "^1.1.0",
+        "http-interop/http-middleware": "^0.4.1"
```

Comment:
You are absolutely right. I will add it to `suggest` and `require-dev`. Tests are on my list too